docs/user-guide/concepts.md

```diff
@@ -352,6 +352,19 @@ overwritten by your actions. You can then rewrite the exception to safely
 exclude those objects, or else delete the conflicting objects to allow them to
 be replaced.
 
+#### (Beta in v1.1) Inclusive propagation
+Inclusive propagation allows you to choose which objects to propagate by using exceptions. 
```

Comment:
This first sentence doesn't really show how inclusive propagation is different from the regular sort. How about this: instead of "inclusive propagation," call it "Opt-in propagation." Then the first sentence changes to:

The `Propagate` mode propagates all objects unless directed to otherwise via a selector. By contrast, opt-in propagation, as set by the `AllowPropagate` mode, doesn't propagate objects _unless_ directed to by a selector. That is, for an object with a selector, its behaviour will be identical in both the `Propagate` and `AllowPropagate` modes; the only difference in behaviours is for objects without a selector.